pages/index.js

```diff
@@ -1,5 +1,5 @@
-import ArtPieces from "../components/ArtPieces/index.js";
+import Spotlight from "@/components/Spotlight/index.js";
 
-export default function Homepage({ pieces }) {
-  return <ArtPieces pieces={pieces} />;
+export default function SpotlightPage({ pieces, artist, image }) {
```

Comment:
we could give Spotlight directly the random image and artist instead of the whole pieces-array

```suggestion
export default function SpotlightPage({ artist, image }) {
```